### PR TITLE
chore(build): Add travis-ci configuration file.

### LINKED ISSUE #1: Build on Travis CI
Add support for building project on Travis CI.

### PR DESCRIPTION
**Pull request by destroyerofbuilds**
Closes #1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  matrix:
+    - NPM_VERSION=2
+    - NPM_VERSION=3
+matrix:
+  fast_finish: true
+node_js:
+  - "0.10"
+  - "0.12"
+  - "4"
+  - "5"
```
Comment (destroyerofbuilds) [inline .travis.yml]: @jarena and @bheiskell I've added Node 5 to the build matrix. Please let me know if that would be an issue given our other policies.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # labeledpipe
 
-[![Build Status](<redacted>)](<redacted>)
+[![Build Status](https://travis-ci.org/factset/labeledpipe.svg?branch=master)](https://travis-ci.org/factset/labeledpipe)
 [![Coverage Status](<redacted>)](<redacted>)
```
Comment (AdamHerrmann) [inline README.md]: Do we get a badge for this from the codecov thing?

Comment (destroyerofbuilds) [inline README.md]: Yes, we get: [codecov.io](https://codecov.io/github/factset/labeledpipe?branch=master)

I was planning on adding it in a follow-up pull request, but given that codecov is part of our Travis CI setup, I would be fine adding it as part of the pull request. What would be your preference?

Comment (AdamHerrmann) [inline README.md]: I'm honestly fine either way, but it seem related enough to this PR that you might as well add it.

Comment (destroyerofbuilds) [inline README.md]: I've added the coverage badge.

**Pull request merged**